include/bluetooth/bluetooth.h

```diff
@@ -436,8 +622,13 @@ struct bt_le_scan_param {
 	/** Scan type (BT_LE_SCAN_TYPE_ACTIVE or BT_LE_SCAN_TYPE_PASSIVE) */
 	u8_t  type;
 
-	/** Bit-field of scanning filter options. */
-	u8_t  filter_dup;
+	union {
+		/** Bit-field of scanning filter options. */
+		u8_t  filter_dup __deprecated;
+
+		/** Bit-field of scanning options. */
+		u32_t options;
```

Comment:
this forced application to use same scan parameters on both phys

Reply:
Yes, I have left this out. Unsure if needed or not.

Reply:
Correct. I have left per-phy scan-parameters (and conn-parameters) out.

Reply:
I think we should give users the option. If necessary through a separate API that sets them similar to the one you had in your second commit of theses series.

Reply:
I have added interval_coded, and window_coded to the params, when set to zero, these will use the same as interval/window.